pyproject.toml

```diff
@@ -84,17 +84,3 @@ exclude_lines = [
     "pragma: py{ignore_python_version}",
     "def _ipython_key_completions_",
 ]
-
-[tool.cibuildwheel]
-# Skip pypy on mac due to numpy/accelerate issues
-skip = ["pp*", "*musllinux*"]
-build-verbosity = 1
-environment = { PIP_PREFER_BINARY=1 }
-test-requires = "pytest scipy"
-test-command = "pytest --pyargs astroscrappy"
```

Comment:
@astrofrog - is it still possibke to keep some options here (e.g. build-verbosity or environment) or is there a way to specify that in the workflow file ?

Reply:
Let's just add them back here (the ones that aren't duplicated in the workflow). Have just pushed a new version here.

Reply:
Ok thanks!